atlassian/confluence.py

```diff
@@ -92,24 +92,23 @@ def get_page_space(self, page_id):
         """
         return ((self.get_page_by_id(page_id, expand='space') or {}).get('space') or {}).get('key')
 
-    def get_page_by_title(self, space, title, start=None, limit=None):
```

Comment:
@andreas-j-hauser  Can we just set into method variables def get_page_by_title(self, space, title, expand=None, start=0, limit=1) ?

Reply:
Yes we can. I see the benefit of your suggestion.

But I'm not so happy with this solution, because using of the parameters brings trouble.
Do you think it helps do not remove parameter start and limit, but ignore the values?
It is possible to mark the parameter as obsolete?

Reply:
About ignore, it will work ok. 
about obsolete, I see only decorator, but it is information as is.
https://stackoverflow.com/questions/2536307/decorators-in-the-python-standard-lib-deprecated-specifically